diff -rupN GT-I9100_Update4-Kernel/arch/arm/mach-exynos/busfreq.c MyKernel/arch/arm/mach-exynos/busfreq.c
--- GT-I9100_Update4-Kernel/arch/arm/mach-exynos/busfreq.c	2012-03-16 06:12:44.000000000 +0100
+++ MyKernel/arch/arm/mach-exynos/busfreq.c	2012-04-02 11:07:50.746573000 +0200
@@ -105,2 +105,2 @@ struct busfreq_table {
-	{LV_0, 400000, 1100000, 0, 0},
-	{LV_1, 267000, 1000000, 0, 0},
+	{LV_0, 400000, 950000, 0, 0},
+	{LV_1, 267000, 875000, 0, 0},
@@ -172,5 +172,5 @@ struct busfreq_table {
-	{1150000, 1050000, 1050000},
-	{1125000, 1025000, 1025000},
-	{1100000, 1000000, 1000000},
-	{1075000, 975000, 975000},
-	{1050000, 950000, 950000},
+	{1050000, 975000, 975000},
+	{1025000, 975000, 975000},
+	{975000, 950000, 950000},
+	{975000, 900000, 900000},
+	{950000, 875000, 875000},
diff -rupN GT-I9100_Update4-Kernel/arch/arm/mach-exynos/cpufreq-4210.c MyKernel/arch/arm/mach-exynos/cpufreq-4210.c
--- GT-I9100_Update4-Kernel/arch/arm/mach-exynos/cpufreq-4210.c	2012-03-16 06:12:44.000000000 +0100
+++ MyKernel/arch/arm/mach-exynos/cpufreq-4210.c	2012-04-02 18:56:25.262591000 +0200
@@ -142,8 +142,8 @@ static const unsigned int asv_voltage_A[
-	 * @200  :
-	 */
-	{ 0, 0, 0, 0, 0, 0, 0, 0 },
-	{ 1350000, 1350000, 1300000, 1275000, 1250000, 1225000, 1200000, 1175000 },
-	{ 1300000, 1250000, 1200000, 1175000, 1150000, 1125000, 1100000, 1075000 },
-	{ 1200000, 1150000, 1100000, 1075000, 1050000, 1025000, 1000000, 975000 },
-	{ 1100000, 1050000, 1000000, 975000, 975000, 950000, 925000, 925000 },
-	{ 1050000, 1000000, 975000, 950000, 950000, 925000, 925000, 925000 },
+	 * @200  :
+	 */
+	{ 0, 0, 0, 0, 0, 0, 0, 0 },
+	{ 1225000, 1225000, 1175000, 1150000, 1125000, 1100000, 1075000, 1050000 },
+	{ 1200000, 1150000, 1100000, 1075000, 1050000, 1025000, 1000000, 975000 },
+	{ 1075000, 1075000, 975000, 975000, 925000, 925000, 900000, 900000 },
+	{ 1000000, 1000000, 950000, 950000, 925000, 900000, 875000, 850000 },
+	{ 950000, 900000, 875000, 850000, 850000, 825000, 825000, 825000 },
--- ../GT-I9100_Update4-Kernel/arch/arm/mach-exynos/mach-u1.c	2012-03-16 06:12:44.000000000 +0100
+++ ./arch/arm/mach-exynos/mach-u1.c	2012-04-24 18:25:23.480769797 +0200
@@ -3075,4 +3075,4 @@
-REGULATOR_INIT(ldo13, "VCC_3.0V_LCD", 3100000, 3100000, 1,
-		REGULATOR_CHANGE_STATUS, 1);
-#else
-REGULATOR_INIT(ldo13, "VCC_3.0V_LCD", 3000000, 3000000, 1,
+REGULATOR_INIT(ldo13, "VCC_3.0V_LCD", 2800000, 2800000, 1,
+		REGULATOR_CHANGE_STATUS, 1);
+#else
+REGULATOR_INIT(ldo13, "VCC_3.0V_LCD", 2700000, 2700000, 1,
@@ -3095,4 +3095,4 @@
-REGULATOR_INIT(ldo18, "TOUCH_LED_3.3V", 3300000, 3300000, 0,
-		REGULATOR_CHANGE_STATUS, 1);
-#else
-REGULATOR_INIT(ldo18, "TOUCH_LED_3.3V", 2500000, 3300000, 0,
+REGULATOR_INIT(ldo18, "TOUCH_LED_3.3V", 3000000, 3000000, 0,
+		REGULATOR_CHANGE_STATUS, 1);
+#else
+REGULATOR_INIT(ldo18, "TOUCH_LED_3.3V", 2500000, 3300000, 0,
@@ -3177,12 +3177,12 @@
 static struct regulator_init_data buck5_init_data = {
 	.constraints	= {
 		.name		= "VMEM_1.2V",
-		.min_uV		= 1200000,
-		.max_uV		= 1200000,
+		.min_uV		= 1100000,
+		.max_uV		= 1100000,
 		.apply_uV	= 1,
 		.always_on	= 1,
 		.state_mem	= {
-			.uV	= 1200000,
+			.uV	= 1100000,
 			.mode	= REGULATOR_MODE_NORMAL,
 			.enabled = 1,
 		},

 

